sample-apps/go-sample-app/main.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"math/rand"
+	"net"
+	"net/http"
+	"time"
+
+	"github.com/aws-otel-commnunity/sample-apps/go-sample-app/collection"
+	"github.com/gorilla/mux"
+	"go.opentelemetry.io/contrib/instrumentation/github.com/gorilla/mux/otelmux"
+	"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
+	"go.opentelemetry.io/otel/metric/global"
+)
+
+// This sample application is in conformance with the ADOT SampleApp requirements spec.
+func main() {
+	ctx := context.Background()
+
+	// The seed for 'random' values used in this applicaiton
+	rand.Seed(time.Now().UnixNano())
+
+	// Client starts
+	shutdown, err := collection.StartClient(ctx)
+	if err != nil {
+		fmt.Println(err)
+	}
+	defer shutdown(ctx)
```

Comment:
```suggestion
	shutdown, err := collection.StartClient(ctx)
	if err != nil {
		log.Fatal(err)
	}
	defer shutdown(ctx)
```

Best to simply exit on error here as `shutdown()` wil be `nil` and a panic will ensue.

Reply:
Done